SU-HSDO/suhumsci
# Local saml

# READY FOR REVIEW

# Summary
- Add composer command to copy necessary saml configs to the correct directory
- Add action link to user list page to add SSO user
- modified stanford_ssp add user page permission.

# Needed By (Date)
- End of sprint 1/23

# Urgency
- High

# Steps to Test

1. checkout this branch
1. composer install --prefer-source
1. Sync a site `blt drupal:sync --site=[sitename]`
1. copy the default local saml files `simplesamlphp/config/default.local.authsources.php` & `simplesamlphp/config/default.local.config.php`
  * Copy the file and name the file without `default.`
1. execute `composer copy-saml` this will copy those local files into the saml directories
1. enable stanford_ssp `drush en stanford_ssp -y`
1. log into the site using your saml login creds
1. view the /admin/user page. 
1. verify "Add SUNetID User"  action link is at the top of the page.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
